pkg/finality-grandpa/logger.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package grandpa
+
+type logLevel int
+
+const (
+	debug logLevel = iota
+	trace
+)
+
+type Logger interface {
```

Comment:
Is the idea here that the default logger is a noop, but a user of the package can call `SetLogger` to use their own logger?

Reply:
Yes that's the idea.